sdk-lib/src/main/java/cash/z/ecc/android/sdk/type/WalletTypes.kt

```diff
@@ -73,7 +73,24 @@ interface UnifiedAddress {
     val rawTransparentAddress: String
 }
 
-enum class ZcashNetwork(val id: Int, val networkName: String, val saplingActivationHeight: Int, val defaultHost: String, val defaultPort: Int) {
+interface ZcashNetwork {
+    val id: Int
+    val networkName: String
+    val saplingActivationHeight: Int
+    val defaultHost: String
+    val defaultPort: Int
+
+    // Required to enable extension functions
+    companion object
+}
+
+enum class NetworkType(
```

Comment:
Note this is a public API change, so we'll need to update the changelog and migration readme for this.  It shouldn't have a huge impact, but it will probably break compile compatibility in our own wallet app.

This seems like a worthwhile API change, I just wanted to call it out explicitly.

Reply:
Thanks for bringing this up. I'll put together that documentation.